_config.yml

```
@@ -30,6 +30,7 @@ seo_keywords: >  # this symbol ignores newlines
   "Stephen Jacobs"
 twitter_account: "@RITMAGIC"
 url: "https://fossrit.github.io"
+open@ritURL: "https://openr.it/"
```

I am not a Jekyll expert but it is probably a good idea to avoid using symbols in variable names.

```suggestion
openRitUrl: "https://openr.it/"
```

---

@Yiroh is this URL referred to in many places? I guess its nice to have this as a variable in case the site changes to using the one on the rit.edu domain, but it would be more compelling if it was actually needed due to linkink to OPEN@RIT from several places

---

but yeah definitely remove the symbol

---

seems like this may also have been intended to cover the link to OPEN@RIT under the "who we are" section of the homepage

---

Yeah so I added that when I was trying to make the tab redirect to Open@RIT in tabs.yml, however I learned you could just type the link in and it'd work as intended so I can safely remove this.